apis/apps/v1alpha1/configconstraint_types.go

```diff
@@ -230,6 +234,12 @@ type TPLScriptTrigger struct {
 	Sync *bool `json:"sync,omitempty"`
 }
 
+type AutoTrigger struct {
+	// processName is process name
+	// +optional
```

Comment:
seems it is a required filed in this api. and pls do `make manifests` if the api file has changed.

Reply:
The "ProcessName" field is optional as database engines automatically reload configuration files. 
Kubeblocks is not required to perform any specific actions in this regard.

Reply:
I included the "omitempty" option for both the "ProcessName" and "AutoTrigger" fields. Afterwards, I ran the commands `make manifests` and `make generate`.

Reply:
ok

Reply:
@wangyelei